Gemfile.lock

```diff
@@ -578,9 +582,8 @@ GEM
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
-    rexml (3.2.5)
-    rollbar (2.19.1)
-      multi_json
+    rexml (3.2.6)
```

Comment:
### [rexml](https://github.com/ruby/rexml)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.2.5 → 3.2.6

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/rexml/3.2.5/3.2.6)_]





<details>
<summary>Commits</summary>

A change of **25** commits. See the full changes on [the compare page](https://github.com/ruby/rexml/compare/v3.2.5...v3.2.6).

These are the first 10 commits:
- (143d244) [Bump version](https://github.com/ruby/rexml/commit/143d244d0bf48fefc8017c25bc0ea7bb98076fce)
- (072b02f) [Set 2.5 as minimum required ruby version for gem (#70)](https://github.com/ruby/rexml/commit/072b02fdcf4993e61cb39f4ed545f77e2f98d3d5)
- (e941ff1) [Document that REXML follows the Ruby maintenance cycle (#71)](https://github.com/ruby/rexml/commit/e941ff17ed3dad428d946b15524bb3529e684266)
- (db12276) [Fix typo in NEWS.md (#72)](https://github.com/ruby/rexml/commit/db12276286f3b44c90727b48b9c5ca8f8e531db3)
- (28ce89f) [Fix typo in NEWS.md (#75)](https://github.com/ruby/rexml/commit/28ce89fd12389a45ee72f46ec10e529f1c1da100)
- (2694bcf) [Tutorial (#77)](https://github.com/ruby/rexml/commit/2694bcf1c743b27ed3394089a0147588eac08f3a)
- (c83774c) [doc: link to tutorial (#78)](https://github.com/ruby/rexml/commit/c83774cff0416c02eef64a31113d2f65990266fa)
- (fc94069) [Fix typos](https://github.com/ruby/rexml/commit/fc94069641019fd7627a0a621032c51a268998d1)
- (d442ccf) [gemspec: Drop unused directives (#83)](https://github.com/ruby/rexml/commit/d442ccf27935b92679264099b751e200cf12b0de)
- (afafbac) [Fix RDoc for Element (#87)](https://github.com/ruby/rexml/commit/afafbacd8a8c1947b63eb0b46d698da76c831d98)


</details>